SI/Sem.02/Pract.02/Task4.cpp

```diff
@@ -0,0 +1,39 @@
+// ConsoleApplicationPrakt2Task4.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+int main()
+{
+    int grade;
+    cout << "Result(%): ";
+    cin>> grade;
+    if (grade >= 90)
+    {
+        cout << "Grade: 6+";
+    }
+    if (grade >= 80 && grade<90)
```

Comment:
The program works correctly, if the number is within range [0, 100] but even if you find the condition you need, you will also check all the other conditions, which can be avoided if you use _else if_. That will spare you some of the extra boolean expressions, too.